README.md

```diff
@@ -144,7 +144,8 @@ If you are using one of the authentication methods above along with managed GitH
 
 ```hcl
 allow_unauthenticated_access = true
-allow_github_webhooks        = true
+allow_unauthenticated_access_path_pattern = "/events" # optional, by default is wildcard "*"
+allow_github_webhooks        = true # or manually set Github CIDRs in whitelist_unauthenticated_cidr_blocks
```

Comment:
if this is set to true, it will add the GitHub CIDRS so this comment is not needed